scalafix-tests/integration/src/test/scala/scalafix/tests/config/GitHubUrlRuleSuite.scala

```diff
@@ -37,6 +37,11 @@ class GitHubUrlRuleSuite extends AnyFunSuite with DiffAssertions {
     "https://raw.githubusercontent.com/someorg/some-repo/master/scalafix/rules/" +
       "src/main/scala/fix/RuleName.scala"
   )
+  check(
+    "github:someorg/some-repo/version.RuleName",
```

Comment:
could we add a test with several segments? like `my.package.RuleName`

Reply:
also, a test with the sha maybe? (it's not documented but I would expect the most verbose/complicated syntax to support all features)